discord/http.py

```diff
@@ -2503,6 +2513,43 @@ def edit_application_info(self, *, reason: Optional[str], payload: Any) -> Respo
         payload = {k: v for k, v in payload.items() if k in valid_keys}
         return self.request(Route('PATCH', '/applications/@me'), json=payload, reason=reason)
 
+    def get_poll_answer_voters(
+        self,
+        channel_id: Snowflake,
+        message_id: Snowflake,
+        answer_id: Snowflake,
+        after: Optional[Snowflake] = None,
+        limit: Optional[int] = None,
+    ) -> Response[poll.PollAnswerVoters]:
+        params = {}
+
+        if after:
+            params['after'] = int(after)
+
+        if limit:
```

Comment:
This is going to make `0` return 25 results instead of erroring, I think this should be an explicit `None` check instead.
```suggestion
        if limit is not None:
```